lib/airport.rb

```
@@ -2,20 +2,21 @@
 
 class Airport
 
-  attr_reader :landed_plane, :landed_planes, :capacity
-  def initialize(capacity=2)
-    @plane
+  attr_reader :landed_planes, :capacity
+
+  def initialize(capacity = 2)
     @capacity = capacity
     @landed_planes = []
   end
+
   def take_off(plane)
     @plane = plane
     raise "Plane has already taken off" unless @landed_planes.include?(plane)
     if weather == "Stormy"
       return "Weather is stormy; plane is unable to take off"
```

I think you can raise this as an error as well, instead of it returning the message! that way the code is more consistent with the other error above it